pravega/src/main/resources/reference.conf

```diff
@@ -0,0 +1,4 @@
+akka.alpakka.pravega {
+  reader.timeout = 5000
+  writer.maximum-inflight-messages=10
+}
```

Comment:
It'd be great if the `reference.conf` included a documented example of each existing setting. We can avoid setting default values by commenting the actual setting but it'll be easier to discover the configuration options for users if we concentrate all settings and docs here.

Take for example: https://github.com/akka/akka-persistence-cassandra/blob/master/core/src/main/resources/reference.conf